classes/class-wc-connect-nux.php

```diff
@@ -83,5 +91,272 @@ public function register_add_service_to_zone_pointer( $pointers ) {
+				'button_text'     => $button_text,
+				'image_url'       => $image_url,
+				'should_show_jp'  => true,
+			);
+
+			$base_location = wc_get_base_location();
+			$country = isset( $base_location['country'] )
+				? $base_location['country']
+				: '';
+			switch ( $country ) {
+				case 'CA':
+					$localized_content = array(
+						'description'     => __( "WooCommerce Shipping is almost ready to go! Once you connect your store you'll be able to show your customers live shipping rates when they check out.", 'woocommerce-services' ),
+					);
+					break;
+			}
+
+			$this->show_nux_banner( wp_parse_args( $localized_content = null, $default_content ) );
+		}
+
+		public function show_banner_after_connection() {
+			if ( ! $this->should_display_nux_notice_on_screen( get_current_screen() ) ) {
+				return;
+			}
+
+			$this->show_nux_banner( array(
+				'title'          => __( 'Setup complete! You can now access discounted shipping rates and printing services' ),
+				'description'    => __( 'When you’re ready, you can purchase discounted labels from USPS, and print USPS labels at home.', 'woocommerce-services' ),
+				'button_text'    => __( 'GET STARTED >', 'woocommerce-services' ),
+				'image_url'      => plugins_url(
+					'assets/images/nux-open-box.jpg', dirname( __FILE__ )
+				),
+				'should_show_jp' => false,
+			) );
+		}
+
+		public function show_nux_banner( $content ) {
+			?>
+			<div class="notice wcs-nux__notice" style="display:flex;">
+				<div class="wcs-nux__notice-logo">
+					<img src="<?php echo esc_url( $content['image_url'] );  ?>">
+				</div>
+				<div class="wcs-nux__notice-content">
+					<h1><?php echo esc_html( $content['title'] ); ?></h1>
+					<p><?php echo esc_html( $content['description'] ); ?></p>
+					<button
+						class="woocommerce-services__connect-jetpack"
+					>
+						<?php echo esc_html( $content['button_text'] ); ?>
+					</button>
+					<?php if ( $content['should_show_jp'] ) : ?>
+						<p>By connecting your site you agree to our fascinating <a href="http://google.com">Terms of Service</a> and to <a>share details</a> with WordPress.com.</p>
+					<?php endif; ?>
+				</div>
+				<?php if ( $content['should_show_jp'] ) : ?>
+					<div class="wcs-nux__notice-jetpack">
+						<img src="<?php echo esc_url(
+							plugins_url( 'assets/images/nux-jetpack-logo.jpg', dirname( __FILE__ ) )
+						); ?>">
+						<p>Powered by Jetpack</p>
+					</div>
+				<?php endif; ?>
+			</div>
+			<?php
+		}
+
+		/**
+		 * Activates Jetpack after an ajax request
+		 */
+		function ajax_activate_jetpack() {
+			check_ajax_referer( 'wcs_nux_notice' );
+
+			$result = activate_plugin( 'jetpack/jetpack.php' );
+
+			if ( is_null( $result ) ) {
+				// The function activate_plugin() returns NULL on success.
+				echo 'success';
+			} else {
+				if ( is_wp_error( $result ) ) {
+					echo esc_html( $result->get_error_message() );
+				} else {
+					echo 'error';
+				}
+			}
+
+			wp_die();
```

Comment:
@jeffgolenski do you think I should use them here? Doesn't seem needed since it's a simple text response.